bugzoo/cli/controllers/container.py

```diff
@@ -0,0 +1,125 @@
+from typing import List, Dict, Optional
+import operator
+
+import cement
+
+from ...core.bug import Bug
+from ...core.source import RemoteSource
+from ...exceptions import NameInUseError
+
+
+class ContainerController(cement.Controller):
+    class Meta:
+        label = 'container'
+        description = 'provision and manage containers'
+        stacked_on = 'base'
+        stacked_type = 'nested'
+        output_handler = 'tabulate'
+        extensions = ['tabulate']
+
+    @cement.ex(
+        help='launches an interactive container for a given bug',
+        arguments=[
+            (['bug'], {'help': 'the name of the bug that should be used'}),
+            (['-v', '--volume'],
+             {'help': 'adds a Docker host-container volume mapping',
+              'dest': 'volumes',
+              'action': 'append',
+              'type': str,
+              'default': []}),
```

Comment:
Yeah, I agree that this is weird. Surprisingly, it's also correct. Since the `action` is `append`, `type` actually gives the type of the items contained within the list, rather than the type of `dest`.